CMakeLists.txt

```diff
@@ -11,16 +11,31 @@ project(bpfilter
 
 include(GNUInstallDirs)
 
-find_package(Doxygen REQUIRED)
+option(BUILD_TESTING "Enable testing" OFF)
+option(BUILD_DOCS "Enable building documentation" OFF)
+
+if(BUILD_DOCS)
+    find_package(Doxygen REQUIRED)
```

Comment:
Move this in `docs/CMakeLists.txt`, and modify `add_subdirectory(docs)` below to:

```cmake
if (WITH_DOCS)
    add_subdirectory(docs)
endif ()
```